### ISSUE TITLE
DeeplapV3 accuracy result for person 

### ISSUE THREAD

**Issue by muna-cs** [pytorch/vision#1901]
Can I know the accuracy of the Deeplap v3-Resnet-101 pre-trained model evaluated on COCO val2017 dataset
for **person** class only?
I searched about it but I did not find the accuracy for each class?

**Comment by fmassa** [comment]
Hi,

Sorry for the delay in replying. Here is the IoU per class

```python
{'__background__': '91.3',
 'aeroplane': '79.2',
 'bicycle': '67.9',
 'bird': '64.6',
 'boat': '50.1',
 'bottle': '50.2',
 'bus': '80.0',
 'car': '59.8',
 'cat': '86.4',
 'chair': '41.6',
 'cow': '73.5',
 'diningtable': '35.8',
 'dog': '76.5',
 'horse': '77.7',
 'motorbike': '78.5',
 'person': '82.9',
 'pottedplant': '38.2',
 'sheep': '78.9',
 'sofa': '58.0',
 'train': '79.1',
 'tvmonitor': '66.1'}
```

So for `'person'`, we have an IoU of 82.9

Let us know if you have further questions.
